src/towncrier/newsfragments/485.misc

```diff
@@ -0,0 +1 @@
+Add workflow_dispatch CI trigger
```

Comment:
```suggestion
```

Misc like these don't need a message (and I believe they aren't rendered either).